src/connector/src/sink/redis.rs

```diff
@@ -44,13 +48,57 @@ pub const VALUE_FORMAT: &str = "value_format";
 #[derive(Deserialize, Debug, Clone, WithOptions)]
 pub struct RedisCommon {
     #[serde(rename = "redis.url")]
-    pub url: String,
+    pub url: Vec<String>,
+}
+pub enum RedisConn {
+    Cluster(ClusterConnection),
```

Comment:
What's the difference between `MultiplexedConnection` and `ClusterConnection`? Can single node connection be seen as a special case of cluster connection that has only one node? Or there is any special reason to use `MultiplexedConnection`, such as better performance.

Reply:
Tried connecting to MultiplexedConnection  with single node, can't get it to run